regtest.sh

```diff
@@ -161,49 +217,38 @@ _run-one-case() {
       -p $p \
       -q $q \
       -f $f \
-      -i $case_dir/case.csv \
-      --out-prefix "$case_dir/case"
+      -i $instance_dir/case.csv \
+      --out-prefix "$instance_dir/case"
 
     banner "Summing bits to get 'counts'"
 
     analysis/tools/sum_bits.py \
       $case_dir/case_params.csv \
-      < $case_dir/case_out.csv \
-      > $case_dir/case_counts.csv
+      < $instance_dir/case_out.csv \
+      > $instance_dir/case_counts.csv
   fi
 
-  banner "Constructing candidates"
-
-  # Reuse demo.sh function
-  print-candidates \
-    $case_dir/case_true_inputs.txt $num_unique_values \
-    $num_additional "$to_remove" \
-    > $case_dir/case_candidates.txt
-
-  banner "Hashing candidates to get 'map'"
-
-  analysis/tools/hash_candidates.py \
-    $case_dir/case_params.csv \
-    < $case_dir/case_candidates.txt \
-    > $case_dir/case_map.csv
-
-  local out_dir=$REGTEST_DIR/${test_case_id}_report
+  local out_dir=${instance_dir}_report
   mkdir --verbose -p $out_dir
 
-  # Input prefix, output dir
-  tests/analyze.R -t "Test case: $test_case_id" "$case_dir/case" $out_dir
+  TIMEFORMAT='Running analyze.R took %R seconds'
```

Comment:
I switched to reporting the AnalyzeRAPPOR timing, adding comments to this effect.